res/type-defs-custom.graphql

```diff
@@ -14,8 +14,6 @@ type Info {
     version: String
     " Server unix time in ms"
     time: Float
-    " Reliable upper boundary for pagination by chain_order field. Before this boundary no additions are possible."
-    reliableChainOrderUpperBoundary: String,
```

Comment:
Why?

Reply:
We don't need to expose this to clients. This is the internal data that must be used by Q-Server internally under the hood.